mb_install.py

```diff
@@ -144,6 +144,9 @@ def mb_install_system(env, source, dest):
     return target
 
 def mb_create_install_target(env):
+    with open('install_manifest.txt', 'w') as fp:
```

Comment:
hmm, I suppose you're right. Ryan was saying there was some problem with it being in obj when trying to access it. Also, obj only exists on some platforms.

@ryanthecoder what was wrong with obj?